<div class="alert alert-block alert-info">

__8/8__ Ananya Sriram

# Donovan Feist

# CH EN 3603

# Homework 5

# Problem 3

Determine an average diffusivity of $H_2S$ in water, using penetration theory, from the data below for
absorption of $H_2S$ into a laminar jet of water at 20°C. Jet diameter=1 cm, jet length=7 cm, and solubility
of $H_2S$ in water is $100 \frac{mol}{m^3}$. Assume the contact time is the time of exposure of the jet.

| Jet Flow Rate, cm^3/s | Rate of Absorption, mol/s x 10^6 |
|-----------------------|----------------------------------|
| 0.143                 | 1.5                              |
| 0.568                 | 3.0                              |
| 1.278                 | 4.25                             |
| 2.372                 | 6.15                             |
| 3.571                 | 7.20                             |
| 5.142                 | 8.75                             |

# Solution

Using film theory, with no bulk flow:

$N_{H_2S}=-c\frac{D_{AB}}{\delta}(x_{H_2S_{fluid}}-x_{H_2S_{gas}})=ck_c(x_{H_2S_{gas}}-x_{H_2S_{fluid}})$.

Applying Higbies penetration theory:

$k_c=2\sqrt{\frac{D_{AB}}{\pi t_c}} \Rightarrow$

$N_{H_2S}=c2\sqrt{\frac{D_{AB}}{\pi t_c}}(x_{H_2S_{gas}}-x_{H_2S_{fluid}})=2\sqrt{\frac{D_{AB}}{\pi t_c}}(c_{H_2S_{gas}}-c_{H_2S_{fluid}}).$

Solving for $D_{AB} \Rightarrow$

$D_{AB}=\pi t_c \left (\frac{N}{2(c_{H_2S_{gas}}-c_{H_2S_{fluid}})} \right )^2$

In [3]:
from numpy import pi,array,mean

d=1 #cm
l=7 #cm
A=pi*d*l #cm2
V=(pi*(d/2)**2)*l #cm3
cg_gas=100*(1/100)**3 #mol/cm3
cg_fluid=0 #mol/cm^3
flowrate=array([0.143,0.568,1.278,2.372,3.571,5.142]) #cm3/s
absorption=10**(-6)*array([1.5,3.0,4.25,6.15,7.20,8.75]) #mol/s
flux=absorption/A #mol/(cm2*s)
tc=V/flowrate #s

Dab=pi*tc*(flux/(2*(cg_gas-cg_fluid)))**2 #cm2/s

print(Dab)

print(mean(Dab))

[1.40484515e-05 1.41473843e-05 1.26191175e-05 1.42369685e-05
 1.29615554e-05 1.32943164e-05]
1.3551298939836563e-05


The average diffusivity of $H_2S$ in water is $\boxed{1.355 \cdot 10^{-5} \frac{cm^2}{s}}$.